src/js/pyodide.ts

```diff
@@ -258,12 +263,15 @@ export async function loadPyodide(
   }
 
   if (API.version !== version) {
-    throw new Error(
-      `\
+    const message = `\
 Pyodide version does not match: '${version}' <==> '${API.version}'. \
 If you updated the Pyodide version, make sure you also updated the 'indexURL' parameter passed to loadPyodide.\
-`,
-    );
+`;
+    if (options.allowVersionMismatch) {
+      console.warn(message);
```

Comment:
I think it is okay to not show warning in this case, as this is probably what the users intended.

Reply:
Ok I will remove this. And another thing, which name do you think is better. I've thought of several name and wonder which one do you think is the most appreciate one

- `allowVersionMismatch`
- `skipVersionCheck`
- `ignoreVersionMismatch`
- `suppressVersionMismatch`
– ...

Not having the knowledge of these words' difference.

Since this is an exposed API and may be better not to change so I'd like to be careful.

Reply:
Maybe `skipVersionCheck`?

Reply:
In `loadPackage`, we have [`checkIntegrity` option](https://github.com/pyodide/pyodide/blob/5fb36a458be37270efecdc9638202a13b62d28f8/src/js/load-package.ts#L283) with default value `true`. So for consistency, maybe we can use a positive term like `checkAPIVersion` and setting it to true by default.

Reply:
I agree with the naming. But we don't have a default value for `options` here:

https://github.com/pyodide/pyodide/blob/9428c6f69e997c59a094538c7638d3e8ebb66284/src/js/pyodide.ts#L191

Reply:
Sorry I just found that here is the default options:

https://github.com/pyodide/pyodide/blob/9428c6f69e997c59a094538c7638d3e8ebb66284/src/js/pyodide.ts#L200-L213

Reply:
The missing ones default to `undefined` / `false`